src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -1306,30 +1298,17 @@ private void VisitActualFunction(ParseNode node, bool isDirect, string directNam
 
             if (isDirect)
             {
-                string functionName = directName;
-
-                string overloadedFunctionName = GetFunctionOverload(functionName, parameterCount);
-                if (options.FuncManager.Exists(overloadedFunctionName)) // if the name is a built-in, then add the "()" after it.
-                    overloadedFunctionName += "()";
-                AddOpcode(new OpcodeCall(overloadedFunctionName));
+                if (options.FuncManager.Exists(directName)) // if the name is a built-in, then add the "()" after it.
+                    directName += "()";
+                AddOpcode(new OpcodeCall(directName));
             }
             else
             {
-                OpcodeCall op = new OpcodeCall(string.Empty) { Direct = false };
+                var op = new OpcodeCall(string.Empty) { Direct = false };
```

Comment:
I find it cleaner because the type is evident from the fact that you are assigning a new copy of OpcodeCall. There is quite a bit of debate among the group on this.